lib/bigcommerce/lightstep/version.rb

```diff
@@ -15,6 +15,6 @@
 #
 module Bigcommerce
   module Lightstep
-    VERSION = '1.3.2'.freeze
+    VERSION = '1.3.3'.freeze
```

Comment:
Can you rebase from master here? We'll want this on `1.3.3.pre` - I generally keep release versioning separate from feature functionality